neqo-transport/src/cid.rs

```diff
@@ -306,11 +298,13 @@ impl ConnectionIdEntry<Srt> {
     }
 
     pub fn is_empty(&self) -> bool {
-        self.seqno == CONNECTION_ID_SEQNO_EMPTY || self.cid.is_empty()
+        self.seqno == ConnectionIdManager::SEQNO_EMPTY || self.cid.is_empty()
     }
 }
 
 impl ConnectionIdEntry<()> {
+    const SEQNO_INITIAL: u64 = 0;
```

Comment:
I think that you need to move this to an `impl<T> ConnectionIdEntry<T> {` with no constraints on `T`.  Otherwise, it won't be visible to other implementations.